src/imgui/ImGuiReverseBar.cc

```diff
@@ -241,24 +301,7 @@ void ImGuiReverseBar::showMenu(MSXMotherBoard* motherBoard)
 							});
 						}
 						if (ImGui::TableNextColumn()) {
-							// As we still want to support gcc-11 (and not require 13 yet), we have to use this method
-							// as workaround instead of using more modern std::chrono and std::format stuff.
-
-							// Convert file_time_type to system_clock::time_point (the standard clock since epoch)
-							auto sctp = std::chrono::time_point_cast<std::chrono::system_clock::duration>(
-									ftime - std::filesystem::file_time_type::clock::now() +
-									std::chrono::system_clock::now());
-
-							// Convert system_clock::time_point to time_t (time since Unix epoch in seconds)
-							std::time_t cftime = std::chrono::system_clock::to_time_t(sctp);
-
-							// Convert time_t to local time (broken-down time in the local time zone)
-							std::tm* local_time = std::localtime(&cftime);
-
-							// Get the local time in human-readable format
-							std::stringstream ss;
-							ss << std::put_time(local_time, "%F %T");
-							ImGui::TextUnformatted(ss.str().c_str());
+							ImGui::TextUnformatted(fileTimeToString(ftime).c_str());
```

Comment:
Detail: this comment already applied to the version before this patch, but I only notice it now.
It's more efficient (and shorter) to leave out the `.c_str()` part.

There are various overloads of `ImGui::TextUnformatted()`.
* The version with a single `const char*` parameter internally calls `strlen()` to calculate the end of the string.
* There's also a Dear ImGui version where you pass 2 `const char*`, the beginning and the end of the string, this avoids the `strlen()` calculation.
* We added two overloads that take a `std::string` or `std::string_view` parameter. These delegate to the version with 2 pointers. Thus also avoiding the `strlen()` calculation.

This comment applies here, but also above.

Reply:
OK, will change it in the last patch (if you don't mind I will not put it in the patch before).